.azure/deploy.yml

```diff
@@ -0,0 +1,110 @@
+# On any tag push, we only need to build and deploy to PyPI.  Only existing
+# commits can be tagged, so they must already have passed CI in the PR process.
+
+trigger:
+  tags:
+    include: ["*"]
+
+variables:
+  # Don't set the twine password globally because it's a secret - it should only
+  # be in the environment for the step that actually runs `twine`.
+  TWINE_USERNAME: qiskit
+
+jobs:
+  - job: 'linux'
```

Comment:
I imagine you've already considered job templates as a possible way to eliminate the duplication in this file. I don't really have a strong preference one way or the other, since IIRC, AzDO templates need to be defined in separate files, which might obscure this pipeline's logic.

Reply:
Yeah, for deploy I thought it wasn't worth bothering - the scripts are all a bit different for the different OSes, so any attempt at abstraction would just confuse things, and the scripts are short enough that I thought it would just be harder to read if I moved them all to separate files.